frontend/src/pages/premium.page.tsx

```diff
@@ -216,12 +216,12 @@ const PremiumPromo: NextPage = () => {
       )}
     </LinkButton>
   ) : (
-    <Button
-      disabled={true}
+    <LinkButton
+      href="/premium/waitlist"
       title={l10n.getString("premium-promo-availability-warning-2")}
     >
-      {l10n.getString("premium-promo-hero-cta")}
-    </Button>
+      {l10n.getString("waitlist-submit-label")}
```

Comment:
I think @peiying2's guidance usually is to use unique strings for every instance of a string, even if the text content in English happens to be the same. For example, while the string in English is "Join the Waitlist", a translation of "Join" might be different depending on the context (e.g. imply submission of a form, which is appropriate in its original context, but not here). And re-using an earlier translation should be easy in Pontoon, as the previous translation will be shown and can be accepted with a single click.

Reply:
I hear that, although there was no design proposed for this, just that Tony requested the CTA strings to be the same throughout the interstitial, therefore I found it reasonable to just copy/paste the strings from the `Plans` component.
<img width="744" alt="image" src="https://user-images.githubusercontent.com/13066134/187943691-d23b91af-4a5a-4c5e-9294-49faa26c0ded.png">

Reply:
Yeah, it's fine that the string value is the same, but I think Peiying requested separate IDs as good practice. Anyway, up to you - I'm just relaying info I heard before, don't know how important it is :)